business/istio_validations.go

```diff
@@ -174,7 +174,7 @@ func (in *IstioValidationsService) GetIstioObjectValidations(namespace string, o
 	case kubernetes.AuthorizationPolicies:
 		authPoliciesChecker := checkers.AuthorizationPolicyChecker{AuthorizationPolicies: rbacDetails.AuthorizationPolicies,
 			Namespace: namespace, Namespaces: namespaces, ServiceList: services, ExportedServiceEntries: exportedResources.ServiceEntries,
```

Comment:
This "ServiceList" is needed here in the context of this work?

Reply:
In the business/istio_validations.go I see other references where both ServiceList and RegistryServices are passed.
I have the same comment on those.